### ISSUE TITLE
Limited ankle dorsiflexion

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5911]
**Preferred term label:** Limited ankle dorsiflexion

**Synonyms**

**Definition (free text, please give PubMed ID)** A reduced range of motion limiting how close the foot can be brought up toward the shin., PMID:27465627

**Parent term (use hpo.jax.org/app)** Limitation of movement at ankles HP:0010505

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
